pkg/apis/rancheros.cattle.io/v1/os.go

```diff
@@ -52,4 +52,5 @@ type ManagedOSImageSpec struct {
 
 type ManagedOSImageStatus struct {
 	fleet.BundleStatus
+	metav1.ObjectMeta `json:"metadata,omitempty"`
```

Comment:
mmm this doesn't feel right, there is some logic issue here if the status spec is returning some events (wouldn't make much sense, no?)

Reply:
ehhh sorry about this, this was some tests that I was doing in check with the comment above, seems to have been pushed with this!